loss.py

```
@@ -27,7 +27,7 @@
 	#------split用于将每一行数据用逗号分割成多个对象-----
     #------取分割后的第0列，转换成float格式后添加到列表中-------
     y.append(float(n.split(',')[0]))
-for num in data1:
+for n in data1:
```

Ruff (0.12.2)

Loop control variable `n` not used within loop body

(B007)